src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -70,33 +70,54 @@ export class DynamicFacetValue implements IDynamicFacetValue {
     return `${l(selectOrUnselect, this.displayValue, resultCount)}`;
   }
 
-  private get rangeAnalyticsMeta() {
-    if (this.facet.facetType === FacetType.specific) {
+  private get isRange() {
```

Comment:
This getter and it's use inside of the class is a sign that we should create a dedicated class range values. It will help keep the code-base clean as we continue growing the Dynamic Facet series. We can do this in a separate PR, but ideally sooner than later

Reply:
I agree